app/scripts/modules/core/pipeline/config/stages/runJob/runJobStage.js

```diff
@@ -16,7 +16,7 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.runJobStage', [
       label: 'Run Job',
       description: 'Runs the previously baked or found image',
       strategyDescription: 'Runs the image specified',
-      key: 'runJob',
+      key: 'run',
```

Comment:
You're right, I think "runJobs" will be better. I was copying "deploy" is to "deployServerGroup" as "run" is to "runJob" to disambiguate the parallel deploy vs ad hoc deploy but that will get us in trouble with "runProcess", if we decide to implement that.